style.css

```diff
@@ -170,6 +166,25 @@ a:active {
   margin: 0;
 }
 
+.footer-contents {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 24px 200px 24px;
```

Comment:
- [ ] Let's make sure we get rid of unwanted space at the bottom after footer content (in desktop view) to match the Figma design

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/155603728-d2f473fe-ab8b-4447-a386-bc22a2aa6ca9.png) | ![image](https://user-images.githubusercontent.com/87689547/155603916-98be9420-972e-41ac-8f57-b887625687a6.png)